geoportal/geoportailv3_geoportal/static-ngeo/js/apps/MainController.js

```diff
@@ -1295,11 +1295,20 @@ const MainController = function(
       theme => {
         if (theme) {
           // set color for custom elements (which is currently not called, since in header of vue app)
-          themeSelectorService.setCurrentThemeColors(theme.name)
+          themeSelectorService.setCurrentThemeColors(theme.name);
+
           // set current theme for legacy components
           this.appTheme_.setCurrentTheme(theme.name);
+
           // set 'data-theme' attribute for legacy component colors
-          document.getElementsByTagName('body')[0].setAttribute('data-theme', theme.name)
+          document.getElementsByTagName('body')[0].setAttribute('data-theme', theme.name);
+
+          // set max zooms (was previously done in CataloggController)
+          this.appTheme_.setThemeZooms(theme);
+
+          try {
+            $scope.$apply(); // Force apply on theme switcher to update theme name
```

Comment:
I left a comment on this in https://github.com/Geoportail-Luxembourg/geoportailv3/pull/3172

Reply:
Yes sorry removing this apply (I checked out the PR #3172 to create this branch)